app.js

```diff
@@ -125,7 +125,7 @@ function get_widgets() {
     "sort": instantsearch.widgets.sortBy({
       container: '#sort-by',
       items: [
-        { label: 'Name', value:'BGG'},
+        { label: 'Name', value:SETTINGS.algolia.index_name},
```

Comment:
Ah good call.

Reply:
Yeah, didn't work for me at first, but it was an easy fix! :)